package.json

```diff
@@ -330,6 +330,7 @@
     "@nx/eslint-plugin": "16.9.0",
     "@nx/express": "16.9.0",
     "@nx/jest": "16.9.0",
+    "@nx/js": "16.9.0",
```

Comment:
Nice to conform this to othe Nx packages. We should then remove `@nrwl/js` because this package is a rename of that package.